pickfirst.go

```diff
@@ -114,6 +115,7 @@ func (b *pickfirstBalancer) UpdateClientConnState(state balancer.ClientConnState
 	}
 
 	if envconfig.PickFirstLBConfig && b.cfg != nil && b.cfg.ShuffleAddressList {
+		addrs = append([]resolver.Address{}, addrs...)
```

Comment:
This is nice. Didn't know this trick before. I would have written this as a two-liner with a `make` and `copy`.